ads/google/ima.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../../src/3p'
+
+/**
+ * Loads the IMA SDK.
+ *
+ * @param {!Window} global
+ * @param {function()} cb
+ */
+function getImaSdk(global, cb) {
+  // video.js player
+  loadScript(global, '//vjs.zencdn.net/5.3/video.min.js', cb);
+  // IMA SDK library
+  loadScript(global, '//imasdk.googleapis.com/js/sdkloader/ima3.js', cb);
+  // video.js contrib-ads plugin (used by IMA plugin)
+  // TODO(sbusolits): Figure out where to host this.
+  loadScript(global, 'http://gvabox.com/sbusolits/amp/js/videojs.ads.js', cb);
```

Comment:
This is a test box we use that doesn't have https support - for full release we'll need to find somewhere else to host these files where we can load them via https.